lib/streamlit/DeltaGenerator.py

```diff
@@ -2014,6 +2037,132 @@ def date_input(self, element, label, value=None, key=None):
         )
         return current_value
 
+    @_with_element
+    def number_input(
+        self,
+        element,
+        label,
+        value=NoValue(),
```

Comment:
Hmmm... this means the API for `st.number_input` will be slightly different from `st.slider()`. I see why you chose to have it this way, and it makes sense. But we _may_ want to change this later. We'll see!

Ok to leave it as is for now